parsers/US_NY.py

```diff
@@ -0,0 +1,136 @@
+#!/usr/bin/env python
+
+"""Real time parser for the state of New York."""
+
+import arrow
+from collections import defaultdict
+from operator import itemgetter
+import pandas as pd
+
+
+ny = arrow.now('America/New_York')
+#Mix csv requires zero padded day.
+ny_date = str(ny.year) + str(ny.month) + (str(ny.day) if ny.day > 9 else str(ny.day).zfill(2))
```

Comment:
This should be defined inside a method (else it will stay constant throughout time)